src/foam/u2/crunch/CrunchController.js

```diff
@@ -556,7 +556,9 @@ foam.CLASS({
 
       intercept.capables.forEach(capable => {
         capable.capabilityIds.forEach((c) => {
-          var seq = this.toGraphAgentWizard(this.createCapableWizardSequence(intercept, capable, c, x));
+          // REVIEW: Launches a capable wizard per capable requirement i.e., capabilityId, should not return to launch point in the loop.
+          var seq = this.createCapableWizardSequence(intercept, capable, c, x);
+          seq = this.toGraphAgentWizard(seq.remove('ReturnToLaunchPointAgent'));
```

Comment:
You can actually do this for all capable wizards in createCapableWizardSequence, if they are launched from the UI they would be transient not capable